### PR TITLE
I have completed all the requirements

### PR DESCRIPTION
**Pull request by omarsalem7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://twitter.com/Omar80491499)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thanks for Microverse for this opportunity.
```
Comment (carlos-ssh) [inline README.md:33]: - Provide a link to your ``License``.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Hello world project.
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
```
Comment (carlos-ssh) [inline README.md:9]: - Provide a description about the features for this project.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Hello world project.
+
+![screenshot](./app_screenshot.png)
```
Comment (carlos-ssh) [inline README.md:7]: - Update your image.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css" />
+
```
Comment (carlos-ssh) [inline index.html:10]: - In your ``PULL REQUEST`` provide a description about the features that you implemented on this branch.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @omarsalem7 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by omarsalem7** [comment]
hello,

I add the changes

**Pull request merged**